contracts/tools/Swaps.sol

```diff
@@ -4,17 +4,18 @@
  */
 pragma solidity ^0.8.0;
 
-import "@openzeppelin/contracts/utils/math/SafeMath.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
-import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 
 import "./ERC1820Client.sol";
 import "../interface/ERC1820Implementer.sol";
 
 import "../extensions/userExtensions/IERC1400TokensRecipient.sol";
+import "../interface/IERC20HoldableToken.sol";
+import "../interface/IHoldableERC1400TokenExtension.sol";
 
-import "../ERC1400.sol";
+import "../IERC1400.sol";
```

Comment:
There are still many places in the comments for example where "DVP" hasn't been renamed "Swap"